src/components/header/style.module.css

```diff
@@ -399,7 +412,6 @@
 	padding: 18px 23px 12px;
 	line-height: var(--header-height);
 	text-align: center;
-	z-index: 100;
```

Comment:
Superfluous `z-index`, it sits on the header (`z-index` of `500` already) and nothing overlaps it either.